docs/reference/api/docker_remote_api_v1.24.md

```diff
@@ -3571,6 +3571,77 @@ Return low-level information on the node `id`
 -   **404** – no such node
 -   **500** – server error
 
+### Remove a node
+
+
+`DELETE /nodes/<id>`
+
+Remove a node [`id`] from the Swarm.
+
+**Example request**:
+
+    DELETE /nodes/24ifsmvkjbyhk HTTP/1.1
+
+**Example response**:
+
+    HTTP/1.1 200 OK
+    Content-Length: 0
+    Content-Type: text/plain; charset=utf-8
+
+**Status codes**:
+
+-   **200** – no error
+-   **404** – no such node
+-   **500** – server error
+
+### Update a node
+
+
+`POST /nodes/<id>/update`
+
+Update the node `id`.
```

Comment:
@vdemeester would you be able to add something about the NodeSpec, or would you prefer to do that in a follow-on PR?